src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -608,18 +608,18 @@ inline CORINFO_InstructionSetFlags EnsureInstructionSetFlagsAreValid(CORINFO_Ins
             resultflags.RemoveInstructionSet(InstructionSet_AVX512F);
         if (resultflags.HasInstructionSet(InstructionSet_AVX512F_VL) && !resultflags.HasInstructionSet(InstructionSet_AVX512F))
             resultflags.RemoveInstructionSet(InstructionSet_AVX512F_VL);
-        if (resultflags.HasInstructionSet(InstructionSet_AVX512CD) && !resultflags.HasInstructionSet(InstructionSet_AVX512F))
```

Comment:
Unintentional change?

Reply:
This change in instruction set order happens when I run src/coreclr/tools/Common/JitInterface/ThunkGenerator/gen.sh even on a clean main branch. I assumed there was some uncommitted change as it's consistent on gen.sh re-runs but I see recent changes in JIT interface don't include it so it must be something in my setup. I'll revert them.